<?xml version="1.0"?>
<rss version="2.0">
    <channel>
        <title>白骨生花 • Posts by &#34;pytorch&#34; tag</title>
        <link>http://qiji5211.com</link>
        <description>同行者，拿起火把！</description>
        <language>zh-CN</language>
        <pubDate>Wed, 07 Feb 2024 17:30:58 +0800</pubDate>
        <lastBuildDate>Wed, 07 Feb 2024 17:30:58 +0800</lastBuildDate>
        <category>大创</category>
        <category>新生</category>
        <category>慧试</category>
        <category>TS</category>
        <category>C语言</category>
        <category>JS</category>
        <category>Vue3</category>
        <category>HTML5</category>
        <category>CSS3</category>
        <category>pytorch</category>
        <item>
            <guid isPermalink="true">http://qiji5211.com/2024/02/07/NER/pytorch%E5%AE%89%E8%A3%85%E7%8E%AF%E5%A2%83%E9%97%AE%E9%A2%98/</guid>
            <title>pytorch安装环境问题</title>
            <link>http://qiji5211.com/2024/02/07/NER/pytorch%E5%AE%89%E8%A3%85%E7%8E%AF%E5%A2%83%E9%97%AE%E9%A2%98/</link>
            <category>pytorch</category>
            <pubDate>Wed, 07 Feb 2024 17:30:58 +0800</pubDate>
            <description><![CDATA[ &lt;h1 id=&#34;自己安&#34;&gt;&lt;a class=&#34;anchor&#34; href=&#34;#自己安&#34;&gt;#&lt;/a&gt; 自己安&lt;/h1&gt;
&lt;p&gt;突然兴起想重装 Anaconda 和 Pycharm, 好家伙，整整两天才给全部整明白，最大的原因见文末，在此记录一下我踩的坑。&lt;/p&gt;
&lt;p&gt;参考【PyTorch 深度学习快速入门教程（绝对通俗易懂！）【小土堆】】 &lt;span class=&#34;exturl&#34; data-url=&#34;aHR0cHM6Ly93d3cuYmlsaWJpbGkuY29tL3ZpZGVvL0JWMWhFNDExdDdSTi8/c2hhcmVfc291cmNlPWNvcHlfd2ViJmFtcDt2ZF9zb3VyY2U9NGJkODUyZjgzNGYyZDlhNmE4Y2UwNjA5YTM2MDAxZDQ=&#34;&gt;https://www.bilibili.com/video/BV1hE411t7RN/?share_source=copy_web&amp;amp;vd_source=4bd852f834f2d9a6a8ce0609a36001d4&lt;/span&gt;&lt;/p&gt;
&lt;p&gt;Anaconda（ &lt;code&gt;Anaconda3-2023.09-0-Windows-x86 64.exe&lt;/code&gt; ）,pycharm，pytorch 已下好且 pytorch（本机没有满足要求的 GPU，选的 CPU）可正常激活，中文路径问题可参考 CSDN 通过配置环境变量解决，仅最后一步不对。&lt;/p&gt;
&lt;figure class=&#34;highlight raw&#34;&gt;&lt;figcaption data-lang=&#34;&#34;&gt;&lt;/figcaption&gt;&lt;table&gt;&lt;tr&gt;&lt;td data-num=&#34;1&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;(pytorch) PS C:\Users\ 王越洋 &amp;gt; python&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;2&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Python 3.6.13 |Anaconda, Inc.| (default, Mar 16 2021, 11:37:27) [MSC v.1916 64 bit (AMD64)] on win32&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;3&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Type &amp;quot;help&amp;quot;, &amp;quot;copyright&amp;quot;, &amp;quot;credits&amp;quot; or &amp;quot;license&amp;quot; for more information.&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;4&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&amp;gt;&amp;gt;&amp;gt; import torch&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;5&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&amp;gt;&amp;gt;&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/figure&gt;&lt;p&gt;但配置时报错如下：&lt;/p&gt;
&lt;figure class=&#34;highlight raw&#34;&gt;&lt;figcaption data-lang=&#34;&#34;&gt;&lt;/figcaption&gt;&lt;table&gt;&lt;tr&gt;&lt;td data-num=&#34;1&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py:21: UserWarning: mkl-service package failed to import, therefore Intel (R) MKL initialization ensuring its correct out-of-the box operation under condition when Gnu OpenMP had already been loaded by Python process is not assured. Please install mkl-service package, see http:&amp;#x2F;&amp;#x2F;github.com&amp;#x2F;IntelPython&amp;#x2F;mkl-service&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;2&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;3&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Traceback (most recent call last):&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;4&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\site-packages\numpy\core\__init__.py&amp;quot;, line 22, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;5&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    from . import multiarray&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;6&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py&amp;quot;, line 21, in do_import&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;7&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;8&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\site-packages\numpy\core\multiarray.py&amp;quot;, line 12, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;9&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    from . import overrides&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;10&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py&amp;quot;, line 21, in do_import&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;11&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;12&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\site-packages\numpy\core\overrides.py&amp;quot;, line 7, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;13&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    from numpy.core._multiarray_umath import (&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;14&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py&amp;quot;, line 21, in do_import&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;15&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;16&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;ImportError: DLL load failed: 找不到指定的模块。&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;17&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;During handling of the above exception, another exception occurred:&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;18&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Traceback (most recent call last):&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;19&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\code.py&amp;quot;, line 91, in runcode&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;20&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    exec (code, self.locals)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;21&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;&amp;lt;input&amp;gt;&amp;quot;, line 1, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;22&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py&amp;quot;, line 21, in do_import&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;23&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;24&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\site-packages\torch\__init__.py&amp;quot;, line 613, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;25&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    _C._initExtension (manager_path ())&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;26&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py&amp;quot;, line 21, in do_import&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;27&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;28&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\site-packages\torch\cuda\__init__.py&amp;quot;, line 685, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;29&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    from . import amp&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;30&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py&amp;quot;, line 21, in do_import&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;31&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;32&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\site-packages\torch\cuda\amp\__init__.py&amp;quot;, line 1, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;33&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    from .autocast_mode import autocast, custom_fwd, custom_bwd  # noqa: F401&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;34&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py&amp;quot;, line 21, in do_import&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;35&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;36&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\site-packages\torch\cuda\amp\autocast_mode.py&amp;quot;, line 5, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;37&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    import numpy as np&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;38&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py&amp;quot;, line 21, in do_import&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;39&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;40&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\site-packages\numpy\__init__.py&amp;quot;, line 140, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;41&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    from . import core&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;42&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;D:\ruanjian\Pycharm\PyCharm 2023.3.2\plugins\python\helpers\pydev\_pydev_bundle\pydev_import_hook.py&amp;quot;, line 21, in do_import&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;43&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    module &amp;#x3D; self._system_import (name, *args, **kwargs)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;44&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  File &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\lib\site-packages\numpy\core\__init__.py&amp;quot;, line 48, in &amp;lt;module&amp;gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;45&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    raise ImportError (msg)&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;46&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;ImportError: &lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;47&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;48&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Importing the numpy C-extensions failed. This error can happen for&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;49&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;many reasons, often due to issues with your setup or how NumPy was&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;50&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;installed.&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;51&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;We have compiled some common reasons and troubleshooting tips at:&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;52&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    https:&amp;#x2F;&amp;#x2F;numpy.org&amp;#x2F;devdocs&amp;#x2F;user&amp;#x2F;troubleshooting-importerror.html&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;53&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Please note and check the following:&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;54&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  * The Python version is: Python3.6 from &amp;quot;C:\Users\ 王越洋 \.conda\envs\pytorch\python.exe&amp;quot;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;55&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  * The NumPy version is: &amp;quot;1.19.2&amp;quot;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;56&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;and make sure that they are the versions you expect.&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;57&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Please carefully study the documentation linked above for further help.&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;58&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Original error was: DLL load failed: 找不到指定的模块。&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/figure&gt;&lt;p&gt;&lt;s&gt;初步怀疑问题在于&lt;/s&gt;（是小白没错了，后面人家说和这个没关系，看到这个报错的 uu 不要惊慌）&lt;/p&gt;
&lt;p&gt;但当时我不知道啊，遂运行提示命令，无果，约半小时后提示某文件没有 admission，CSDN 有文指出该问题可以重新更新 conda，下载一小时没下完。两个都运行了两次左右，各种重装 Anaconda，结果人家输出我没有 admission，如果你和我一样装 Anaconda 时选的 &lt;code&gt;给所有人装&lt;/code&gt; 的选项，记得在权限里设置 &lt;code&gt;所有人可修改&lt;/code&gt; 即可解决。当然，不设置也可以。&lt;/p&gt;
&lt;p&gt;最后，这个错无伤大雅，重点不在这，不要运行 &lt;code&gt;conda update -n base -c defaults conda&lt;/code&gt;  就是了。&lt;/p&gt;
&lt;figure class=&#34;highlight raw&#34;&gt;&lt;figcaption data-lang=&#34;&#34;&gt;&lt;/figcaption&gt;&lt;table&gt;&lt;tr&gt;&lt;td data-num=&#34;1&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;(pytorch) PS C:\Users\ 王越洋 &amp;gt; conda install pytorch torchvision torchaudio cpuonly -c pytorch&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;2&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Collecting package metadata (current_repodata.json): done&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;3&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Solving environment: done&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;4&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;5&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;6&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&amp;#x3D;&amp;#x3D;&amp;gt; WARNING: A newer version of conda exists. &amp;lt;&amp;#x3D;&amp;#x3D;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;7&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  current version: 23.7.4&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;8&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;  latest version: 24.1.0&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;9&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;10&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Please update conda by running&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;11&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;12&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;    $ conda update -n base -c defaults conda&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;13&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;14&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;Or to minimize the number of packages updated during conda update use&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;15&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;16&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;     conda install conda&amp;#x3D;24.1.0&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;17&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td data-num=&#34;18&#34;&gt;&lt;/td&gt;&lt;td&gt;&lt;pre&gt;# All requested packages already installed.&lt;/pre&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/figure&gt;&lt;p&gt;这个报错的意思是你有库安了，但是没安全。&lt;/p&gt;
&lt;h1 id=&#34;深夜寻求某宝&#34;&gt;&lt;a class=&#34;anchor&#34; href=&#34;#深夜寻求某宝&#34;&gt;#&lt;/a&gt; 深夜寻求某宝&lt;/h1&gt;
&lt;p&gt;凌晨 00:23，解决无果，遂面向某宝解决（cost20￥），次日早，某阿里员工给我一波各种操作，两小时后告诉我问题是 numpy 库没安全，然后各种调试，找版本，然后安，解决！&lt;/p&gt;
&lt;p&gt;下午，感觉自己又行了，于是自己重新建了一个虚拟环境，把人家建的删了，报错依旧，重装 numpy 无果，&lt;s&gt;发疯发疯&lt;/s&gt;，感觉这个人解决问题不能一劳永逸（其实是我没学会。。。人家还是很厉害滴）~&lt;/p&gt;
&lt;h1 id=&#34;晚上寻求拼夕夕&#34;&gt;&lt;a class=&#34;anchor&#34; href=&#34;#晚上寻求拼夕夕&#34;&gt;#&lt;/a&gt; 晚上寻求拼夕夕🌟&lt;/h1&gt;
&lt;p&gt;已是晚 10：20, 继第 10 次左右重新安装调试后，随便捞了个能安的人（cost15￥），看得出来他也很焦灼，甚至还出现了新错误:&lt;/p&gt;
&lt;p&gt;(1) Environment 创建失败（这个和库没关系，是说你的源或者网络问题，建议换源即可）&lt;/p&gt;
&lt;p&gt;(2) 系统找不到指定路径。这个就比较棘手了，人家配了环境变量，各种调试，无果。通过装 &lt;code&gt;Anaconda3-2020.11 Windows-x86 64.exe&lt;/code&gt; ，再来一遍 pip 配置，成功！！！&lt;/p&gt;
&lt;p&gt;通过 pip 安装就不会像通过 conda 安装报错，成功解决！！！&lt;/p&gt;
&lt;p&gt;小问题：conda 安装的虚拟环境位置在 &lt;code&gt;C:/User/XXX/.conda/env/&lt;/code&gt;  下的，但这种方式安装的虚拟环境位置在 Anaconda3 的安装目录下，其他运行均正常，虽然还是会报版本不一致的错，但无碍。本次配置终于完成！！&lt;/p&gt;
&lt;h1 id=&#34;结果&#34;&gt;&lt;a class=&#34;anchor&#34; href=&#34;#结果&#34;&gt;#&lt;/a&gt; 结果&lt;/h1&gt;
&lt;p&gt;综上，出现此类错误的人建议装低版本一点的 Anaconda (卸的时候记得用自带的工具卸，写完检查环境变量删干净没)，安装时记得把 PATH 选项勾上（要不然要自己配，容易配错），最新版的 Anaconda 应该是有小毛病（库不全），&lt;span class=&#34;red&#34;&gt;谨慎安装！！！&lt;/span&gt;&lt;/p&gt;
&lt;p&gt;祝看到这里的各位 PyTorch 学习顺利！🌟&lt;/p&gt;
 ]]></description>
        </item>
    </channel>
</rss>
